### ISSUE TITLE
Making the Write Page responsive

### ISSUE THREAD

**Issue by amrit03b** [amrit03b/TravelTales#42]
Make the Write Page responsive such that it fits all screen sizes.

**Comment by GauravGupta993** [comment]
I would like to work on this as well.

**Comment by amrit03b** [comment]
> I would like to work on this as well.

Assigned.

**Comment by GauravGupta993** [comment]
@amrit03b  I have made it responsive such that the section showing Write Your Travel Journey and section showing Your TravelTales appear side by side for all device sizes. A horizontal scroll bar appears for Your TravelTales section in case size of device is small. Navbar is already responsive. Creating a PR for it still do i need to do anything else ?

**Comment by amrit03b** [comment]
> @amrit03b  I have made it responsive such that the section showing Write Your Travel Journey and section showing Your TravelTales appear side by side for all device sizes. A horizontal scroll bar appears for Your TravelTales section in case size of device is small. Navbar is already responsive. Creating a PR for it still do i need to do anything else ?

Hey, that's all you need to do.

**Comment by amrit03b** [comment]
@GauravGupta993 Actually, the older version was more responsive as in smaller screen it is difficult to Write if both the sections are side by side.

**Comment by GauravGupta993** [comment]
> @GauravGupta993 Actually, the older version was more responsive as in smaller screen it is difficult to Write if both the sections are side by side.

@amrit03b  What changes do i need to make ? Do I need to increase the size of Your TravelTales section for smaller devices?

**Comment by amrit03b** [comment]
> > @GauravGupta993 Actually, the older version was more responsive as in smaller screen it is difficult to Write if both the sections are side by side.
> 
> @amrit03b  What changes do i need to make ? Do I need to increase the size of Your TravelTales section for smaller devices?

Yes, increasing the size of "Your TravelTales" would be good enough for now. Also make the background image cover both the sections "Write Your Travel Journal" and "Your TravelTales "
